tests/mocks/pve7x/nodes.go

```diff
@@ -782,7 +782,7 @@ func nodes() {
         "uptime": 2501631,
         "kversion": "Linux 5.15.108-1-pve #1 SMP PVE 5.15.108-2 (2023-07-20T10:06Z)",
         "cpuinfo": {
-            "mhz": "3400.000",
+            "mhz": 3400,
```

Comment:
I saw the latest commit which is changing the type from String to Int so need to update here as well but I don't believe this is related to what I have proposed on the current PR.

Reply:
so the guy that made that change says it's an int now in pve8, maybe change to StringOrInt? tried to get that guy to fix it in his latest pr but he closed it

Reply:
Hi @luthermonson, I just tried to follow up with the latest updates to bring in a proper fix. To be perfectly honest I wasn't aware of that issue until I do `go test`.

Reply:
my apologies, i merged it before realizing the consequences in the testing.